vm/src/import.rs

```diff
@@ -144,3 +144,35 @@ fn find_source(vm: &VirtualMachine, current_path: PathBuf, name: &str) -> Result
         None => Err(format!("No module named '{}'", name)),
     }
 }
+
+pub fn remove_importlib_frames(vm: &VirtualMachine, exc: &PyObjectRef) -> PyObjectRef {
+    let always_trim = objtype::isinstance(exc, &vm.ctx.exceptions.import_error);
+
+    if let Ok(tb) = vm.get_attribute(exc.clone(), "__traceback__") {
+        if objtype::isinstance(&tb, &vm.ctx.list_type()) {
+            let mut tb_entries = objsequence::get_elements_list(&tb).to_vec();
+            tb_entries.reverse();
+            let mut new_tb = Vec::with_capacity(tb_entries.len());
+
+            for tb_entry in tb_entries.iter() {
+                let mut current_chunk = vec![];
+                let location_attrs = objsequence::get_elements_tuple(&tb_entry);
+                let file_name = objstr::get_value(&location_attrs[0]);
+                if file_name != "_frozen_importlib" && file_name != "_frozen_importlib_external" {
+                    current_chunk.clear();
+                    new_tb.push(tb_entry.clone())
+                } else {
+                    current_chunk.push(tb_entry.clone());
+                    let run_obj_name = objstr::get_value(&location_attrs[0]);
```

Comment:
This is the same as `file_name` earlier on.

Reply:
Nice catch. Thanks.